### PR TITLE
TSD to Typings

### PR DESCRIPTION
**Pull request by Joeper214**
ref: https://skyarch.backlog.jp/view/NC-666

## Update process

``` sh
cd YOUR_SKYHOPPER_DIRECTORY
cd frontend/
npm i
gulp type
gulp ts
cd ..
```

### CODE REVIEW

**Comment by Joeper214** [comment]
Please review and Merge PR. @root-beer

File: frontend/src/helper.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ Vue.component('bootstrap-tooltip', {
     },
   },
   template: `
-<span data-toggle="tooltip" data-placement="{{placement}}" data-original-title="{{title}}">
+<span data-toggle="tooltip" :data-placement="{{placement}}" :data-original-title="{{title}}">
```
Comment (root-beer) [inline frontend/src/helper.ts]: Tooltips don't appear. (Vue version is 1.0.26)

File: frontend/tsconfig.json [hunk 2]
```diff
@@ -28,6 +24,10 @@
     "src/models/s3_bucket.ts",
     "src/models/serverspec.ts",
     "src/models/snapshot.ts",
+    "src/project_parameters.ts",
+    "src/server_status.ts",
+    "src/serverspec-gen.ts",
+    "src/users_admin.ts"
```
Comment (root-beer) [inline frontend/tsconfig.json]: missing trailing comma

File: frontend/src/helper.ts (hunk 1 above)
Comment (Joeper214) [inline frontend/src/helper.ts]: Thank you for reviewing. I'll find out the `tooltip` problems tomorrow.

File: frontend/tsconfig.json (hunk 2 above)
Comment (Joeper214) [inline frontend/tsconfig.json]: I guess `tsconfig.json` file is formatted this way. Did you encountered this error when running `gulp type` and `gulp ts`?

Comment (root-beer) [inline frontend/tsconfig.json]: I encountered this.
```
% gulp ts
[18:16:02] Using gulpfile ~/work/projects/skyhopper/frontend/gulpfile.js
[18:16:02] Starting 'ts'...
Failed to parse file 'tsconfig.json': Unexpected string in JSON at position 785.
node_modules/typings-core/node_modules/typescript/lib/lib.d.ts(136,5): error TS1131: Property or signature expected.
node_modules/typings-core/node_modules/typescript/lib/lib.d.ts(136,14): error TS1005: ';' expected.
...
```
But I appended a comma, the problem is solved.

**Comment by root-beer** [comment]
LGTM! Thanks. :bow:

**Pull request merged**